src/lib/menu/menu-directive.ts

```diff
@@ -196,15 +196,23 @@ export class MatMenu implements AfterContentInit, MatMenuPanel<MatMenuItem>, OnI
    */
   @Input('class')
   set panelClass(classes: string) {
+    if (this.previousPanelClass && this.previousPanelClass.length) {
```

Comment:
`this.previousPanelClass` is repeated in a few places. Move it into a variable so it's a bit more concise and it minifies better.